Triangulation/examples/Triangulation/CMakeLists.txt

```diff
@@ -30,7 +32,9 @@ if(TARGET CGAL::Eigen3_support)
     regular_triangulation
     triangulation
     triangulation_data_structure_dynamic
-    triangulation_data_structure_static)
+    triangulation_data_structure_static
+    triangulation1
```

Comment:
@mglisse can we rename `triangulation1.cpp` to `triangulation1.h` as it gets `#include`ed by `triangulation.cpp

Reply:
I changed the file extension. Instead of including one  might also have three source files for one target.
As it is now, it is not an exemplary example.

Reply:
Sure, ok to rename them, or even just inline that code in triangulation.cpp and remove the 2 other files.
(I was never super familiar with the Triangulation package)